pkgs/development/python-modules/betterproto/default.nix

```diff
@@ -63,13 +70,14 @@ buildPythonPackage rec {
   '';
 
   meta = with lib; {
-    description = "Clean, modern, Python 3.6+ code generator & library for Protobuf 3 and async gRPC";
+    description = "Code generator & library for Protobuf 3 and async gRPC";
```

Comment:
This is the upstream description. Not sure why we would want to change that.

Reply:
The "marketing part" or "how upstream thinks the project looks like" usually doesn't really add a lot of value to the description. If the library is modern and clean then we would not need to disable tests :wink: 

Sure, this change is highly opinionated.